### ISSUE TITLE
Missing conformance image: k8s.gcr.io/conformance:v1.21.0-beta.1

### ISSUE THREAD

**Issue by AlekSi** [kubernetes/kubernetes#100584]
#### What happened:

```
$ docker pull k8s.gcr.io/conformance:v1.21.0-beta.1
Error response from daemon: manifest for k8s.gcr.io/conformance:v1.21.0-beta.1 not found: manifest unknown: Failed to fetch "v1.21.0-beta.1" from request "/v2/conformance/manifests/v1.21.0-beta.1".
```

#### What you expected to happen:

Image pulled successfully, like it does for `k8s.gcr.io/conformance:v1.21.0-beta.0`.

#### How to reproduce it (as minimally and precisely as possible):

```
docker pull k8s.gcr.io/conformance:v1.21.0-beta.1
```

**Comment by neolit123** [comment]
the missing image for this particular tag was caused by a bug.
discussion here indicated that its not trivial to re-release it:
https://kubernetes.slack.com/archives/CJH2GBF7Y/p1616523838082500

**Comment by BenTheElder** [comment]
This image should be available in the newly released `v1.21.0-rc.0` (we had more fun getting it working), FWIW.

**Comment by AlekSi** [comment]
Thank you, `k8s.gcr.io/conformance:v1.21.0-rc.0` will do for us.
